Yoast/yoast-components
# Added color value for no-index

## Summary

This PR can be summarized in the following changelog entry:

* Adds `"$color_noindex":"#1e8cbe"` to `colors.json` so that the bullets for noindexed posts will be blue instead of the default grey. 

## Test instructions

This PR can be tested by following these steps:

* Set one or more posts to noindex (Allow search engines to show this Post in search results?-->No)
* In your Yoast SEO post overview (In dashboard) the color of bullets labeled "Posts that should not show up in search results" or "Posts that are set to “noindex” should be blue instead of grey. 

Fixes #446
